src/pages/ReportDetailsPage.js

```diff
@@ -133,7 +133,7 @@ class ReportDetailsPage extends Component {
                         >
                             <View style={styles.mb4}>
                                 <RoomHeaderAvatars
-                                    avatarIcons={OptionsListUtils.getAvatarSources(this.props.report)}
+                                    avatarIcons={OptionsListUtils.getReportIcons(this.props.report)}
```

Comment:
```suggestion
                                    reportIcons={OptionsListUtils.getReportIcons(this.props.report)}
```

Reply:
Ah, this suggestion changes the property name of the`RoomHeaderAvatars` component, and we're already using the property name `avatarIcons` consistently in other Avatar similar components. Is ok if I change `avatarIcons` to `reportIcons` to make it consistent on all the similar components? Or maybe just to `icons`?

Reply:
Yeah, there would definitely be some cascading effects for renaming this. I would be very happy to call everything `icons` and keep it consistent across all components that it touches.